KeychainExample/App.tsx

```diff
@@ -44,33 +45,36 @@ export default class KeychainExample extends Component {
     username: '',
     password: '',
     status: '',
-    biometryType: null,
-    accessControl: null,
-    securityLevel: null,
-    storage: null,
+    biometryType: undefined,
+    accessControl: undefined as undefined | Keychain.ACCESS_CONTROL,
+    securityLevel: undefined as undefined | Keychain.SECURITY_LEVEL,
+    storage: undefined as undefined | Keychain.STORAGE_TYPE,
+    selectedStorageIndex: 0,
+    selectedSecurityIndex: 0,
+    selectedAccessControlIndex: 0,
   };
 
   componentDidMount() {
```

Comment:
Could consider refactoring this one to a functional component at some point.

Reply:
Yes, that is the plan eventually :D